sklearn/model_selection/tests/test_search.py

```diff
@@ -332,6 +333,33 @@ def test_grid_search_groups():
         gs.fit(X, y)
 
 
+def test_grid_search_future_warnings():
+    # Test that warnings are raised. Will be removed in 0.22
+
+    X = np.arange(100).reshape(10, 10)
+    y = np.array([0] * 5 + [1] * 5)
+
+    clf = LinearSVC(random_state=0)
+    grid = {'C': [.1]}
+
+    def init(clf, parameters, return_train_score):
+        grid_search = GridSearchCV(clf, parameters,
+                                   return_train_score=return_train_score)
+        grid_search.fit(X, y)
+        return grid_search
+
+    msg = "Computing training scores may affect performance "
```

Comment:
This is not doing what you think it does: msg only contains the first line. Please use parentheses to make sure msg has the full message with all the lines concatenated.